integration/conftest.py

```diff
@@ -231,52 +235,43 @@ def introducer(reactor, tahoe_venv, temp_dir, flog_gatherer, request):
 tub.location = tcp:localhost:9321
 '''.format(log_furl=flog_gatherer)
 
-    intro_dir = join(temp_dir, 'introducer')
-    print("making introducer", intro_dir)
-
-    if not exists(intro_dir):
-        mkdir(intro_dir)
-        done_proto = _ProcessExitedProtocol()
-        _tahoe_runner(
-            done_proto,
-            reactor,
-            tahoe_venv,
-            request,
-            (
-                'create-introducer',
-                '--listen=tcp',
-                '--hostname=localhost',
-                intro_dir,
-            ),
+        intro_dir = join(temp_dir, 'introducer')
+        print("making introducer", intro_dir)
+
+        if not exists(intro_dir):
+            mkdir(intro_dir)
+            done_proto = _ProcessExitedProtocol()
+            _tahoe_runner(
+                done_proto,
+                reactor,
+                tahoe_venv,
+                request,
+                (
+                    'create-introducer',
+                    '--listen=tcp',
+                    '--hostname=localhost',
+                    intro_dir,
+                ),
+            )
+            pytest_twisted.blockon(done_proto.done)
+
+        # over-write the config file with our stuff
+        with open(join(intro_dir, 'tahoe.cfg'), 'w') as f:
+            f.write(config)
+
+        # on windows, "tahoe start" means: run forever in the foreground,
+        # but on linux it means daemonize. "tahoe run" is consistent
+        # between platforms.
```

Comment:
the code that is directly responsible for choosing a tahoe subcommand seems to be somewhere else now.  also "tahoe start" has been removed.  Maybe this comment has outlived its usefulness.

Reply:
Yep. The comment was moved there, but this instance wasn't removed (there were originally two copies of this comment).